test-cases/manager/manager-multiple-restores-schema-and-data.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 2100
+
+round_robin: true
+
+instance_type_db: 'i3en.3xlarge'
+instance_type_loader: 'c5.2xlarge'
+nemesis_class_name: NodeTerminateAndReplace
+
+region_name: us-east-1
```

Comment:
Add a comment here that the region is specified because the backup bucket is in the same region.